gNMI_Interface/Subscribe.resource

```diff
@@ -1,11 +1,46 @@
 *** Settings ***
 Documentation    Resources specific to gNMI Subscribe operation.
-Library          SubscribeLibrary.py  ${LIB_CONFIG}
 Resource         gNMIClient.resource
 
 
 *** Keywords ***
 
-Verify Subscribe
-    [Arguments]  ${mode}
-    Test Subscribe  ${mode}  ${GET-PATH}
+Subscribe ${mode} to default path with default encoding
+    Given subscription paths    ${GET-PATH}
+    Then subscribe    ${mode}    ${lib_config.default_encoding}
+    And close subscription
+
+Subscription ${mode} with encoding ${encoding}
+    Subscribe    ${mode}    ${encoding}
+
+Subscription ${mode} with default encoding
+    Subscription ${mode} with encoding ${lib_config.default_encoding}
+
+Device responds
+    Check updates    ${SUBSCRIPTION-TIMEOUT}
+
+Subscribe ONCE with supported encodings
+    @{encodings}=    last supported encodings
+    FOR    ${encoding}    IN    @{encodings}
+        Given subscription ONCE with encoding ${encoding}
+        Then device responds
+        Close subscription
+    END
+
+Device capabilities
```

Comment:
Well, yes and no. The problem is that `Capabilities.resources` cannot be imported since it imports its library, that would not work. (But note that `SubscribeLibrary` inherits from `CapabilitiesLibrary`, so the actual code is reused.)